src/routes/health/health.router.ts

```diff
@@ -1,26 +1,10 @@
 import { Router } from 'express';
-import { HEALTH_ENDPOINT } from '../../constants/endpoint';
-import { getRoutes } from './health.service';
 
-export const router: Router = Router();
+const router: Router = Router();
 
-// getStatus
 router.get('/', (req, res) => {
-  res.status(200).send();
-});
-
-router.get(HEALTH_ENDPOINT, (req, res) => {
   res.status(200).send({
     status: 'API Server is running...',
   });
 });
-
-// getRoutes
-router.get('/routes', (req, res) => {
-  const routes = getRoutes();
-
-  res.status(200).send({
-    numberOfRoutes: routes.length,
-    routes: routes,
-  });
-});
```

Comment:
Refactor broke this so I removed it. We should provide a swagger for docs instead.